app.js

```diff
@@ -0,0 +1,152 @@
+    name: 'Profesional Art Printing Data More',
+    name1: 'Website Protfolio',
+    description: "A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard",
+    technologies: ['html', 'bootstrap', 'Ruby'],
+
+  },
+  {
+    url1: 'assets/images/watermarker.png',
+    url: 'assets/images/watermarker.png',
+    id: 'forth-container',
+    idBtn: 'notsee3',
+    name: 'Profesional Art Printing Data More',
+    name1: 'Profesional Art Printing Data More',
+    description: "A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard",
+    technologies: ['html', 'bootstrap', 'Ruby'],
+
+  },
+  {
+    url1: 'assets/images/watermarker.png',
+    url: 'assets/images/Blurwatermark.png',
+    id: 'fifth-container',
+    idBtn: 'notsee4',
+    name: 'Profesional Art Printing Data More',
+    name1: 'Data Dashboard Healthcare',
+
+    description: "A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard",
+    technologies: ['html', 'bootstrap', 'Ruby'],
+
+  },
+  {
+    url1: 'assets/images/watermarker.png',
+    url: 'assets/images/3watermark.png',
+    id: 'sixth-container',
+    idBtn: 'notsee5',
+    name: 'Profesional Art Printing Data More',
+    name1: 'Website Protfolio',
+    description: "A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard",
+    technologies: ['html', 'bootstrap', 'Ruby'],
+
+  },
+
+];
+
+function renderCards(i) {
+  const div = document.createElement('div');
+  div.className = 'middle-1-container1';
+
+  div.innerHTML = `<div id="${projects[i].id}" class="middle-1-container1">
+        <h2 class="mobile">${projects[i].name}</h2>
+      <h3 class="desktop">${projects[i].name1}</h3>
+        
+        
+        <p>${projects[i].description}</p>
+  
+        <ul class="cardbtn">
+              ${projects[i].technologies.map((tech) => `<li><button type="button" id="${projects[i].idSmallbtn}" class="middle-1-container1-btn">${tech}</button></li>`).join('')}
+        </ul>
+        <button type="button" id="${projects[i].idBtn}" onclick="openPopup()" class="see-button-one1">See project</button>
+      </div>`;
+  // eslint-disable-next-line prefer-const
+  let nameid = projects[i].id;
+
+  // eslint-disable-next-line prefer-template
+  div.querySelector('#' + nameid).style.backgroundImage = `url("${projects[i].url}")`;
+  document.querySelector('.middle-section').appendChild(div);
+}
+
+// eslint-disable-next-line no-unused-vars
+function openPopup() {
+  const pop = document.querySelector('.popup2');
+  // eslint-disable-next-line quotes
+  pop.style.display = "flex";
+  // eslint-disable-next-line no-var
+  const blur = document.getElementById('middlesec');
+  blur.classList.toggle('active');
+}
+
+// eslint-disable-next-line no-unused-vars
+function closePopup() {
```

Comment:
- [ ] Good job implementing the popup functionality, but unfortunately I am not able to close the popup modal after opening it, kindly add functionality to close `X` the popup

![close](https://user-images.githubusercontent.com/80895497/163488969-47affd1f-fded-4715-a642-a36909610446.PNG)

Reply:
- Good job fixing this 👍